src/seedu/addressbook/Main.java

```diff
@@ -112,7 +112,7 @@ private CommandResult executeCommand(Command command)  {
             storage.save(addressBook);
             return result;
         } catch (Exception e) {
-            ui.showToUser(e.getMessage());
+             ui.showToUser(e.getMessage());
```

Comment:
Unintentional change? The indentation change causes it to violate the coding standard.